force-app/main/default/triggers/OpportunityTrigger_deleteCloseWonOpp.trigger

```diff
@@ -0,0 +1,8 @@
+trigger OpportunityTrigger_deleteCloseWonOpp on Opportunity (before delete) {
+    for (Opportunity opp : Trigger.old) {
+        // Check if the opportunity is closed-won and related account's industry is Banking
+        if (opp.StageName == 'Closed Won' && opp.Account.Industry == 'Banking') {
+            opp.addError('Cannot delete closed opportunity for a banking account that is won');
+        }
+    }
```

Comment:
Did this pass the unit itests? The related fields like Account.Industry are not available in Trigger context and should need to be queried